src/Microsoft.Framework.PackageManager/Program.cs

```diff
@@ -256,6 +257,37 @@ public int Main(string[] args)
                 });
             });
 
+            app.Command("list", c =>
+            {
+                c.Description = "Print the dependencies of a given project.";
+                var coreclrRoot = c.Option("--coreclr",
```

Comment:
humm may need a better name. I'm a bit hesitated on "runtime". First of all it is not used in the case of CLR framework. Second, it is actually used to point the CoreCLR packages folder rather than the runtime folder. And also put "runtime" is a little bit ambiguous, in `kvm` it accepts a runtime version name.